ci/scripts/run-test-network-basic.sh

```diff
@@ -0,0 +1,60 @@
+set -euo pipefail
+
+FABRIC_VERSION=${FABRIC_VERSION:-2.2}
+CHAINCODE_LANGUAGE=${CHAINCODE_LANGUAGE:-javascript}
+CHAINCODE_NAME=${CHAINCODE_NAME:-basic}
+
+function print() {
+	GREEN='\033[0;32m'
+  NC='\033[0m'
+  echo
+	echo -e "${GREEN}${1}${NC}"
+}
+
+print "Creating network"
+  ./network.sh up createChannel -ca -s couchdb -i "${FABRIC_VERSION}"
+print "Deploying ${CHAINCODE_NAME} chaincode"
+./network.sh deployCC -ccn "${CHAINCODE_NAME}" -ccv 1 -ccs 1 -ccl "${CHAINCODE_LANGUAGE}" -cci initLedger
+
+
+# Run Javascript application
+print "Initializing Javascript application"
+pushd ../asset-transfer-basic/application-javascript
+rm -rf wallet
+npm install
+print "Executing enrollAdmin.js"
+node enrollAdmin.js
+print "Executing registerUser.js"
+node registerUser.js
```

Comment:
no need for
```
node enrollAdmin.js
node registerUser.js
```

since app.js calls above to add users.

Reply:
I like this where CI just makes one call to the application under test. It will decide what it needs to do.

Reply:
I didn't realize `app.js` was importing and calling enroll and register, removed the direct calls to enroll and register